### PR TITLE
Fix endpoint DSL build

### PR DESCRIPTION
**Pull request by omarsmak**
Somehow after the new endpointdsl improvements, the build was failing due to some components weren't built while the endpointdsl module was built. I just re-built the components first and ran endpointdsl to correctly build the DSL for the endpoints

### CODE REVIEW

File: core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/ServiceNowEndpointBuilderFactory.java [hunk 1]
```diff
@@ -1034,7 +1034,7 @@ default AdvancedServiceNowEndpointBuilder timeFormat(String timeFormat) {
 
     public interface ServiceNowBuilders {
         /**
-         * ServiceNow (camel-servicenow)
+         * ServiceNow (@@@ARTIFACTID@@@)
```
Comment (oscerd) [inline core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/ServiceNowEndpointBuilderFactory.java]: This is really weird.

Comment (omarsmak) [reply core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/ServiceNowEndpointBuilderFactory.java]: I think is something else, not from the endpointdsl MOJO, looks like the generated `json` for this component has this `@@@ARTIFACTID@@@`

Comment (oscerd) [reply core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/ServiceNowEndpointBuilderFactory.java]: Did you rebuild everything without fastinstall and -DskipTests?

Comment (omarsmak) [reply core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/ServiceNowEndpointBuilderFactory.java]: Nope I built it with fastinstall, could be that?

Comment (oscerd) [reply core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/ServiceNowEndpointBuilderFactory.java]: There were a lot of changes yesterday so I would go with a full rebuild in this case.

**Comment by davsclaus** [comment]
Closing as it should build now. Sometimes you need to do a clean install build when there are too many changes, like they did in the endpointdsl recently.

**Comment by oscerd** [comment]
even with a full rebuild servicenow still reports artifactid @davsclaus @omarsmak @lburgazzoli @gnodet

**Comment by omarsmak** [comment]
I think I figured out what is the problem, will shoot a PR fix shortly

**Comment by oscerd** [comment]
thanks @omarsmak

**Pull request closed without merging**